deployments/helm/configmapfiles/discovery-engine/conf.yaml

```diff
@@ -35,10 +35,11 @@ observability:
   system-observability: true
   network-observability: false
   write-logs-to-db: false
+  write-to-db: false

```

Comment:
We have `write-logs-to-db` flag in the above line. What it is used for? How different it is from `write-to-db`?

Reply:
It is used to write kubearmor system logs to SQL! We can make both of these to a single flag!

Reply:
@wazir-ahmed Yes Wazir. This sounds great.
So under observability we will have 2 sub categories 
1. Summary-jobs for publisher ?

Reply:
> It is used to write kubearmor system logs to SQL! We can make both of these to a single flag!

@stefin9898 Do not merge them both into a single flag. Those need to be handled separately.

Reply:
Not properly addressed. Please change the config format as mentioned above.
```yaml
observability: 
  ...
  summary-jobs:
    publisher: true
    write-summary-to-db: true
    cron-interval: "0h1m00s"
```